docs/tutorials/10_torch_qgan.ipynb

```diff
@@ -0,0 +1,752 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "# PyTorch qGAN Implementation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "Description\n",
+    "\n",
+    "adapted from [PyTorch GAN](https://github.com/eriklindernoren/PyTorch-GAN/blob/master/implementations/gan/gan.py)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 43,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Necessary imports\n",
+    "\n",
+    "import numpy as np\n",
+    "from typing import Union, List, Optional, Iterable\n",
+    "\n",
+    "from torch import Tensor, stack, reshape\n",
+    "from torch.utils.data import DataLoader\n",
+    "from torch.autograd import Variable\n",
+    "from torch.optim import Adam\n",
+    "import torch.nn as nn\n",
+    "import torch.nn.functional as F\n",
+    "\n",
+    "\n",
+    "from qiskit import Aer, QuantumCircuit\n",
+    "from qiskit.utils import QuantumInstance, algorithm_globals\n",
+    "from qiskit.opflow import Gradient, StateFn\n",
+    "from qiskit.circuit.library import TwoLocal\n",
+    "from qiskit.circuit import ParameterExpression, ParameterVector\n",
+    "from qiskit_machine_learning.neural_networks import CircuitQNN\n",
+    "from qiskit_machine_learning.connectors import TorchConnector\n",
+    "from qiskit_machine_learning.datasets.dataset_helper import discretize_and_truncate\n",
+    "\n",
+    "# Set seed for random generators\n",
+    "algorithm_globals.random_seed = 42\n",
+    "np.random.seed = 42"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "### Load training data\n",
+    "\n",
+    "For testing purposes, we decide for a 2D multivariate normal distribution.\n",
+    "Each dimension is represented by 2 qubits."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 44,
+   "outputs": [],
+   "source": [
+    "data_dim = [3, 3]\n",
+    "\n",
+    "training_data = np.random.default_rng().multivariate_normal(mean=[0., 0.], cov=[[1, 0], [0, 1]], size=1000, check_valid='warn',\n",
+    "                                                        tol=1e-8, method='svd')\n",
```

Comment:
A couple of notes:
1. for the sake of consistency with other qiskit tutorials, you should use `algorithm_globals.random` instead `np.random` (algorithm_globals is basically a wrapper over np in this case)
2. no need to add the `default_rng()` part or set arguments that are there by default. 
My suggestion is to do the following:

```suggestion
    "training_data = algorithm_globals.random.multivariate_normal(mean=[0., 0.],  cov=[[1, 0], [0, 1]], size=1000)\n",
```